js/src/components/experience-rating-banner/index.js

```diff
@@ -9,39 +10,55 @@ import { external as externalIcon } from '@wordpress/icons';
  * Internal dependencies
  */
 import AppButton from '../app-button';
+import FeedbackModal from './feedback-modal';
 import './index.scss';
 
 const ExperienceRatingBanner = () => {
-	const handleClick = () => false;
+	const [ showModal, setShowModal ] = useState( false );
+
+	const handleClick = () => {
+		// Open the feedback modal
+		setShowModal( true );
+	};
+
+	const handleRequestClose = () => {
+		// Close the feedback modal
+		setShowModal( false );
+	};
 
 	return (
-		<Notice
-			className="gla-experience-rating-banner"
-			status="info"
-			isDismissible={ true }
-		>
-			<p className="gla-experience-rating-banner__text">
-				{ __(
-					'How was your experience with Google for WooCommerce?',
-					'google-listings-and-ads'
-				) }
-			</p>
-
-			<div className="gla-experience-rating-banner__actions">
-				<AppButton onClick={ handleClick } isSecondary>
-					{ __( 'Good', 'google-listings-and-ads' ) }
-				</AppButton>
-
-				<AppButton
-					isSecondary
-					href="https://woocommerce.com"
-					target="_blank"
-				>
-					{ __( 'Need help', 'google-listings-and-ads' ) }
-					<Icon icon={ externalIcon } size={ 18 } />
-				</AppButton>
-			</div>
-		</Notice>
+		<div className="gla-experience-rating-banner__container">
+			{ showModal && (
+				<FeedbackModal onRequestClose={ handleRequestClose } />
+			) }
+			<Notice
+				className="gla-experience-rating-banner"
```

Comment:
@ankitrox I see we are adding this class name only for the E2E tests. Is there not an alternative way of writing the tests without using the class name?

Reply:
@asvinb  We have a bit of css based on this class name. We can keep it as of now, if there are no serious concerns regarding it.

Reply:
🤦 Apologies @ankitrox . I meant the `gla-experience-rating-feedback-modal` class. I don't see any styles attached to it and using it only for the E2E tests.

Reply:
Sure, removed it.